server/bin/pbench-index.py

```diff
@@ -137,20 +159,7 @@ def main(options, name):
         idxctx.templates.dump_templates()
         return 0
 
-    _re_idx = "RE-" if options.re_index else ""
-    if options.index_tool_data:
-        # The link source and destination for the operation of this script
-        # when it only indexes tool data.
-        linksrc = "TO-INDEX-TOOL"
-        linkdest = "INDEXED"
-    else:
-        # The link source and destination for the operation of this script
-        # when it indexes run, table-of-contents, and result data.
-        linksrc = f"TO-{_re_idx}INDEX"
-        linkdest = "TO-INDEX-TOOL"
-    # We only ever use a symlink'd error destination for indexing
-    # problems.
-    linkerrdest = "WONT-INDEX"
+    # sd_obj = LinkSrcDestObj(options)
```

Comment:
Did you intend to leave this commented? If so, please explain why. We shouldn't be submitting commented code without a persuasive TODO comment explaining why it's commented and under what conditions we expect it to be restored.